datalad_dataverse/remote.py

```diff
@@ -34,6 +75,9 @@ def __init__(self, *args):
         self._url = None
         self._api = None
         self._token = None
+        self._dataset = None
+        self._files_all = None
+        self._files_latest = None
```

Comment:
I have an intuition what `url`, `api`, `token`, and `dataset` are, but it have difficulties to imagine what distinguishes `files_all` and `files_latest`. A comment would be good.